app/src/main/java/com/teamagam/gimelgimel/app/map/esri/MapDragEventsEmitterTouchListenerDecorator.java

```diff
@@ -0,0 +1,112 @@
+package com.teamagam.gimelgimel.app.map.esri;
+
+import android.view.MotionEvent;
+import android.view.View;
+import com.esri.android.map.MapOnTouchListener;
+import com.esri.android.map.MapView;
+import com.teamagam.gimelgimel.app.map.MapDragEvent;
+import com.teamagam.gimelgimel.domain.map.entities.geometries.PointGeometry;
+import io.reactivex.functions.BiFunction;
+import io.reactivex.subjects.Subject;
+
+class MapDragEventsEmitterTouchListenerDecorator implements View.OnTouchListener {
```

Comment:
I disagree... It should implement the interface and not extend some concrete implementation.
It overrides these method to consume its event and not activate some default behavior by the listener (it must do so).